core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala

```diff
@@ -338,7 +338,7 @@ private[spark] class TaskSchedulerImpl private[scheduler](
     }.getOrElse(offers)
 
     // Randomly shuffle offers to avoid always placing tasks on the same set of workers.
```

Comment:
you can remove this comment now that it's duplicated in the shuffleOffers method description